src/pages/Settings/Settings.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Notification from "@canonical/react-components/dist/components/Notification/Notification";
+
+import Header from "components/Header/Header";
+import Layout from "components/Layout/Layout";
+
+import "./settings.scss";
+
+export default function Settings() {
+  return (
+    <Layout>
+      <Header>
+        <span className="l-content settings__header">Settings</span>
```

Comment:
This should be a H1 element

Reply:
I have to override almost every style on the H1 to make it look good. Without it's like...

<img width="226" alt="Screen Shot 2020-02-14 at 11 20 49 AM" src="https://user-images.githubusercontent.com/532033/74553013-12203280-4f1c-11ea-9448-8af0168abea1.png">